BIOS.md

```diff
@@ -1,6 +1,6 @@
 # BIOS Configuration
 
-BIOS version: `F11`
+BIOS version: `F12j`
```

Comment:
Does this mean that now we can do the CFG Lock directly on the BIOS? 🥇

Reply:
@gbasile yes.. with f12j, it's been added in the BIOS GUI..
![20201031_185711](https://user-images.githubusercontent.com/1284948/99155108-07cfd680-26f0-11eb-9f13-71d78b3fb746.jpg)

Reply:
Yep, apparently they added this option in F12j. I already updated to F12j and the CFG Lock option works great.